aeat.py

```diff
@@ -942,10 +920,26 @@ class SIIReportLine(ModelSQL, ModelView):
     invoice_operation_key = fields.Function(
         fields.Selection(OPERATION_KEY, 'SII Operation Key'),
         'get_invoice_operation_key')
-    exemption_key = fields.Char('Exemption Cause', readonly=True)
+    exemption_cause = fields.Char('Exemption Cause', readonly=True)
     aeat_register = fields.Text('Register from AEAT Webservice', readonly=True)
     sii_header = fields.Text('Header')
 
+    @classmethod
+    def __register__(cls, module_name):
+        cursor = Transaction().connection.cursor()
+        table = cls.__table_handler__(module_name)
+        sql_table = cls.__table__()
+
+        exist_sii_excemption_key = table.column_exist('exemption_key')
+
+        super(SIIReportLine, cls).__register__(module_name)
+
+        if exist_sii_excemption_key:
+            # Don't use UPDATE FROM because SQLite nor MySQL support it.
+            cursor.execute(*sql_table.update([sql_table.exemption_cause],
```

Comment:
Better use rename. Ie: https://hg.tryton.org/modules/sale_opportunity/file/tip/opportunity.py#l138

Reply:
Done. I didn't know the rename function.